src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -60,6 +60,30 @@ const en = ({ isDivorce, userCase, partner }: CommonContent) => ({
     You will receive an email by ${userCase.dueDate} after your application has been checked.
     This will have the time, date and court your conditional order will be pronounced.`,
   },
+  subHeading1:
+    userCase.coClarificationUploadDocuments || userCase.coClarificationResponses
+      ? 'Latest information'
+      : 'What you need to do',
+  clarificationSubmitted: {
+    withDocuments: {
+      line1: `You have provided the information requested by the court. You'll receive an email by ${dayjs(
+        userCase.dateSubmitted
+      )
+        .add(16, 'days')
+        .format('D MMMM YYYY')} after the court has reviewed it.`,
+      line2: 'This was the court’s feedback, explaining the information which was needed:',
+      line3: userCase.coRefusalClarificationAdditionalInfo,
+    },
+    withoutDocuments: {
+      subHeading1: 'What you need to do',
+      line1: `You or your ${partner} need to post the documents requested by the court:`,
+      line2:
+        '<strong>HMCTS Divorce and Dissolution Service</strong><br>' + 'PO Box 13226<br>' + 'HARLOW<br>' + 'CM20 9UG',
+      line3: 'This is the feedback the court gave, which explains what documents you need to send:',
+      line4: userCase.coRefusalClarificationAdditionalInfo,
```

Comment:
this is used twice in the same way, i suggest remove one and make it a common use property instead of subproperty